### DISCUSSION TITLE
Comments on "Install Preline UI with React using Tailwind CSS"

### DISCUSSION THREAD

**Discussion by A-Berton** [htmlstreamofficial/preline#462] [Show and tell]
I couldn't make Preline work with my React / Vite & Typescript configuration, the reinitialization helper resulted in a blank page. But this seemed to work for me: 

`import React from 'react';
import ReactDOM from 'react-dom/client';
import './index.css';
import App from './App';
import { BrowserRouter } from 'react-router-dom';

const root = ReactDOM.createRoot(document.getElementById('root')!);
root.render(
  <React.StrictMode>
    <BrowserRouter>
    <App />
    </BrowserRouter>
  </React.StrictMode>
);`

**Comment by olegpix** [comment]
@A-Berton Hi,
please check the React + Vite [demo](https://stackblitz.com/edit/dev-preline-2-vite-react?file=src%2FApp.tsx).

**Comment by langscot** [comment]
Make sure if you are using Dependency Optimisation with Vite (https://vitejs.dev/config/dep-optimization-options), that your `content` array in `tailwind.config.js` is properly pointing to the optimized `preline` dependency JavaScript file.
